core/modules/dashboard/includes/block.taxonomy.inc

```diff
@@ -0,0 +1,154 @@
+<?php
+/**
+ * @file
+ *
+ * Dashboard "Taxonomy" block. Shows statistics and links to manage terms in
+ * vocabularies on the site.
+ */
+class DashboardTaxonomyBlock extends Block {
+
+  /**
+   *  Sets block subject on block view.
+   */
+  function getTitle() {
+    return !empty($this->settings['title']) ? check_plain($this->settings['title']) : t('Categories');
+  }
+
+  /**
+   * 'Admin title' callback for the content type.
+   */
+  function getAdminTitle() {
+    return t('Dashboard manage taxonomy');
+  }
+
+  /**
+   * 'Admin info' callback for the content type.
+   */
+  function getAdminPreview() {
+    return t('Links to administer taxonomy.');
+  }
+
+  /**
+   * Run-time rendering of the body of the block.
+   */
+  function getContent() {
+    if (!module_exists('taxonomy')) {
+      return;
+    }
+
+    $vids = (isset($this->settings['vids'])) ? $this->settings['vids'] : array();
+
+    $vocabs = taxonomy_get_vocabularies();
+    $options = array('destination' => 'admin/dashboard');
```

Comment:
Does that assume that this block will only be placed in a/the Dashboard? If people place it in other dashboards in different paths, then editing the block and saving it would redirect them to the "default" dashboard that ships with core instead of the one they were initially using.

Reply:
hmm, good point. we should use the current path here.

Reply:
fixed in lastest push.